src/components/actions/scheduler.ts

```diff
@@ -40,11 +48,29 @@ export const triggerScheduledActions = async () => {
   DBConnect.triggerScheduledActions()
 }
 
+const checkFile = async (filePath: string) => {
+  if (await DBConnect.checkIfInFileTable(filePath)) {
+    fs.unlink(filePath, function (err) {
+      if (err) throw err
+      else console.log(`Deleted file at ${filePath}`)
+    })
+  }
+}
+
 export const cleanUpFiles = async () => {
   console.log(
     DateTime.now().toLocaleString(DateTime.DATETIME_SHORT_WITH_SECONDS),
-    'Cleaning up preview files...'
+    'Cleaning up filess and file records...'
   )
+  await crawlFileSystem(basePath, checkFile)
   const deleteCount = await DBConnect.cleanUpFiles()
   if (deleteCount > 0) console.log(`${deleteCount} files removed.`)
+  else console.log('no files removed.')
+}
+
+// Manually launch cleanup with command
+if (isManualCleanup) {
+  cleanUpFiles()
 }
```

Comment:
When running `yarn cleanup`, the script never exits. That's because it missing the exit instruction, like this one:
![Screen Shot 2022-10-03 at 4 32 45 PM](https://user-images.githubusercontent.com/5456533/193496832-9b13e0f6-4a61-495d-809b-bc682d49a639.png)